src/sentry/api/endpoints/project_metrics_extraction_rules.py

```diff
@@ -81,62 +88,86 @@ def post(self, request: Request, project: Project) -> Response:
         if not self.has_feature(project.organization, request):
             return Response(status=404)
 
-        rules_update = request.data.get("metricsExtractionRules")
+        config_update = request.data.get("metricsExtractionRules")
 
-        if not rules_update or len(rules_update) == 0:
+        if not config_update:
             return Response(
                 status=400,
                 data={"detail": "Please specify the metric extraction rule to be created."},
             )
-
         try:
-            state_update = self._generate_updated_rule_objects(rules_update)
-            persisted_rules = create_metrics_extraction_rules(project, state_update)
-            updated_rules = serialize(
-                persisted_rules, request.user, MetricsExtractionRuleSerializer()
+            configs = []
+            with transaction.atomic(router.db_for_write(SpanAttributeExtractionRuleConfig)):
+                for obj in config_update:
+                    configs.append(
+                        SpanAttributeExtractionRuleConfig.from_dict(obj, request.user.id, project)
+                    )
+                schedule_invalidate_project_config(
+                    project_id=project.id, trigger="span_attribute_extraction_configs"
+                )
+
+        except Exception:
```

Comment:
Do you want any errors captured here or are failures always user error?

Reply:
Good point, I added a call to capture_exception()